src/plugin/peripheral/nodes/peripheralclusternode.ts

```diff
@@ -6,13 +6,15 @@
  ********************************************************************************/
 
 import * as vscode from 'vscode';
-import { PeripheralBaseNode, ClusterOrRegisterBaseNode } from './basenode';
+import { PeripheralBaseNode, ClusterOrRegisterBaseNode, PERIPHERAL_ID_SEP } from './basenode';
```

Comment:
I see a little bit of inconsistency in file names. I wonder if it should be renamed to `peripheral-cluster-node`